cmd/plugin/node_server.go

```diff
@@ -78,7 +78,7 @@ func (n NodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublishV
 		return
 	}
 
-	// to trigger CI
+	// to trigger CI again
```

Comment:
@vadasambar You might consider using `allow-empty`. You'll not need to remove any unnecessary lines later.
```
git commit --allow-empty --message "🤖 Trigger CI run"
```

Reply:
This is helpful. Thank you Mriyam.